examples/albert/schedule.py

```diff
@@ -9,6 +9,7 @@
 
 from slapo import init_empty_weights
 from typing import Optional
+from slapo.pattern import call
```

Comment:
nit: Not sure if the term `call` would be too common. Maybe `call_pattern` or `call_node`. I don't know.

Reply:
How about just calling it `call_module`? Since we can only match module node now

Reply:
Sounds good to me.